polls/views.py

```diff
@@ -25,12 +25,12 @@ def get(self, request):
         }
         return render(request, self.template_name, context)
 
-class TaskFlowView(LoginRequiredMixin, UserPassesTestMixin, View):
+class TaskFlowView(LoginRequiredMixin, View):
```

Comment:
Why are you removing the mixin? Maybe you need a new mixin?

Reply:
This is builtin mixin and is used to direct to 403 only in case test fails